configure

```diff
@@ -35,7 +35,7 @@ usage_with_default()
 default_coptflags()
 {
     if [ "$1" = 0 ]; then
-	echo "-Og"
+	echo ""
```

Comment:
This is a change?  We used to build -Og if !DEVELOPER, which is a compromise, but a nice one...

Reply:
I don't remember the exact issue, but at least one compiler on Launchpad was complaining about this flag. Let me see if I can find it again, otgherwise I'm happy to revert.

Reply:
Damn, I hit apply automatically :(

I'll do another PR which reverts this, sorry...